src/java.base/share/classes/java/text/ChoiceFormat.java

```diff
@@ -373,7 +404,7 @@ public double[] getLimits() {
     }
 
     /**
-     * Get the formats passed in the constructor.
+     * Get the formats of this ChoiceFormat.
```

Comment:
Same here

Reply:
Fixed, as well as in the other instance, I have also updated https://cr.openjdk.org/~jlu/api/java.base/java/text/ChoiceFormat.html to reflect the changes. Thanks for the review